packages/tinacms/src/admin/pages/CollectionListPage.tsx

```diff
@@ -95,85 +96,117 @@ const CollectionListPage = () => {
                                     </span>
-                                  </span>
-                                </Link>
-                              </td>
-                              <td className="px-6 py-4 whitespace-nowrap">
-                                <span className="block text-xs text-gray-400 mb-1 uppercase">
-                                  Extension
-                                </span>
-                                <span className="h-5 leading-5 block text-sm font-medium text-gray-900">
-                                  {document.node.sys.extension}
-                                </span>
-                              </td>
-                              <td className="px-6 py-4 whitespace-nowrap">
-                                <span className="block text-xs text-gray-400 mb-1 uppercase">
-                                  Template
-                                </span>
-                                <span className="h-5 leading-5 block text-sm font-medium text-gray-900">
-                                  {document.node.sys.template}
-                                </span>
-                              </td>
-                            </tr>
-                          ))}
-                        </tbody>
-                      </table>
-                    </div>
-                  )}
+                                  </td>
+                                  <td className="px-6 py-4 whitespace-nowrap">
+                                    <span className="block text-xs text-gray-400 mb-1 uppercase">
+                                      Extension
+                                    </span>
+                                    <span className="h-5 leading-5 block text-sm font-medium text-gray-900">
+                                      {document.node.sys.extension}
+                                    </span>
+                                  </td>
+                                  <td className="px-6 py-4 whitespace-nowrap">
+                                    <span className="block text-xs text-gray-400 mb-1 uppercase">
+                                      Template
+                                    </span>
+                                    <span className="h-5 leading-5 block text-sm font-medium text-gray-900">
+                                      {document.node.sys.template}
+                                    </span>
+                                  </td>
+                                  {livesiteRoute && (
+                                    <td className="px-6 py-4 whitespace-nowrap">
+                                      <span className="block text-xs text-gray-400 mb-1 uppercase">
+                                        Site Link
+                                      </span>
+                                      <a href={livesiteRoute}>Preview</a>
+                                    </td>
+                                  )}
```

Comment:
Here's where the `Preview` link is being displayed, assuming the `routeMapping` returns a `string`.